apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -136,6 +136,10 @@
             "type": "boolean",
             "default": false
         },
+        "helpLink": {
+          "description": "Link to help source for the assessment.",
```

Comment:
A better explanation here would be better. In particular, referencing the fact that this is a link used by students to get help from staff.

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/7929/commits/a7ab952dc04d980f681aa8bcf73f04727f74ac7e